chrome/content/zotero/bindings/itembox.xml

```diff
@@ -1975,8 +1976,22 @@
 						default:
 							throw ("Invalid transform mode '" + mode + "' in zoteroitembox.textTransform()");
 					}
-					this._setFieldValue(label, newVal);
-					this._modifyField(label.getAttribute('fieldname'), newVal, this.saveOnEdit);
+					
+					var currentFieldName = label.getAttribute('fieldname');
+					// Creator field (e.g., creator-2-lastName)
+					if (currentFieldName.startsWith('creator-')) {
+						let row = Zotero.getAncestorByTagName(document.popupNode, 'row');
+						let creatorIndex = parseInt(currentFieldName.split('-')[1]);
+						let fields = this.getCreatorFields(row);
+						fields[currentFieldName.split('-')[2]] = newVal;
+						this.modifyCreator(creatorIndex, fields);
+						this.item.save();
+					}
+					// Single field (e.g., title)
+					else {
```

Comment:
This is AFAIK <a href="http://en.wikipedia.org/wiki/Indent_style#Variant:_Stroustrup">Variant Stroustrup</a> and not <a href="http://en.wikipedia.org/wiki/Indent_style#Variant:_1TBS">1TBS</a> as wished in https://www.zotero.org/support/dev/client_coding/coding_guidelines . Anyway, if you want it differently, feel free to change it afterwards.